python/whylogs/api/whylabs/session/session_manager.py

```diff
@@ -61,7 +62,9 @@ def is_active() -> bool:
         return SessionManager.get_instance() is not None
 
 
+@deprecated_alias(session_type="allow_anonymous")
```

Comment:
This isn't perfect as it doesn't check/message about `allow_local`. But allegedly we only care about this case :)